gulpfile.js

```diff
@@ -94,29 +95,39 @@ gulp.task('imgmin', function () {
       .pipe(gulp.dest('public/img'));
 });
 
-// webpack build:
-gulp.task('webpack', function() {
-  webpack({
-    entry: './js/main.js',
-    output: {
-      path: __dirname + '/public/js',
-      filename: 'j.js'
-    },
-    plugins: [
-      new webpack.optimize.UglifyJsPlugin({
+// wrapper for JS build, supporting both
+// minified and unminified versions
+function jsBuild(minify) {
+  return gulp.src('js/main.js')
+    .pipe(webpackStream({
+      entry : {
+        j : './js/main.js'
```

Comment:
Having this `entry` section isn't strictly necessary, but it gives you support for having multiple builds (for different types of pages, e.g.) in the future.